resource/reactivedrop_english.txt

```diff
@@ -260,10 +260,10 @@
 "asw_difficulty_carnage"	"Carnage"
 "asw_difficulty_uber"	"Über"
 "asw_difficulty_hardcore"	"Hardcore"
-"asw_difficulty_insane_desc"	"If your squad has finished the campaign on Hard, try Insane for a real challenge! The Swarm move at lightning speeds, do much more damage, and are harder than ever to kill. Prepare for even more swarms of aliens!"
-"asw_difficulty_hard_desc"	"If you’ve played Alien Swarm before, then start here. The Swarm move faster and are harder to kill. Squad tactics are critical to your survival. You’ll also earn more XP."
-"asw_difficulty_normal_desc"	"If you’re new to Alien Swarm, then Normal is a good place to hone your skills. The Swarm will give you a good fight."
-"asw_difficulty_easy_desc"	"If you are new to games, then Easy is the setting for you. The Swarm run more slowly and are easier to kill. You'll have a fighting chance."
+"asw_difficulty_insane_desc"	""
+"asw_difficulty_hard_desc"	""
+"asw_difficulty_normal_desc"	""
+"asw_difficulty_easy_desc"	""
```

Comment:
These should be fully removed rather than blanking them. (Just delete the english ones and run the sync tool and it should do the rest).